README.md

```diff
@@ -26,6 +26,100 @@
 ❗️ To file an issue, please open a ticket on <a href="https://github.com/allenai/allennlp/issues/new/choose">allenai/allennlp</a> and tag it with "Models". ❗️
 </div>
 
+## About
+
+This repository contains the components - such as [`DatasetReader`](https://docs.allennlp.org/main/api/data/dataset_readers/dataset_reader/#datasetreader), [`Model`](https://docs.allennlp.org/main/api/models/model/#model), and [`Predictor`](https://docs.allennlp.org/main/api/predictors/predictor/#predictor) classes - for applying [AllenNLP](https://github.com/allenai/allennlp) to a wide variety of NLP [tasks](#tasks-and-components).
+It also provides an easy way to download and use [pre-trained models](#pre-trained-models) that were trained with these components.
+
+### Tasks and components
+
+This is an overview of the tasks supported by the AllenNLP Models library along with the corresponding components provided, organized by category. For a more comprehensive overview, see the [AllenNLP Models documentation](https://docs.allennlp.org/models/main/) or the [Paperswithcode page](https://paperswithcode.com/lib/allennlp).
+
+- [**Classification**](https://github.com/allenai/allennlp-models/tree/main/allennlp_models/classification)
```

Comment:
Perhaps add a description of what Classification is, like you've done for the other categories, e.g. prediction of discrete integer labels for examples

Reply:
Yeup, good call. I was being lazy 😜